managed/services/supervisord/supervisord.go

```diff
@@ -519,6 +525,21 @@ func (s *Service) addPostgresParams(templateParams map[string]interface{}) {
 	templateParams["PostgresSSLCertPath"] = s.pgParams.SSLCertPath
 }
 
+func (s *Service) addClusterParams(templateParams map[string]interface{}) {
+	templateParams["HAEnabled"] = s.haParams.Enabled
+	if s.haParams.Enabled {
+		templateParams["GrafanaGossipPort"] = s.haParams.GrafanaGossipPort
+		templateParams["HAAdvertiseAddress"] = s.haParams.AdvertiseAddress
+		nodes := make([]string, len(s.haParams.Nodes))
+		for i, node := range s.haParams.Nodes {
+			nodes[i] = fmt.Sprintf("%s:%d", node, s.haParams.GrafanaGossipPort)
```

Comment:
`net.JoinHostPort`

Reply:
`net.JoinHostPort` requires additional conversion, not sure if it's better